source/Cosmos.Common/ByteToString.cs

```diff
@@ -8,7 +8,7 @@ static public class ByteToString
     {
         public static byte[] StrToByteArray(string str)
         {
-            if (str.Length == 0)
+            if (str.Length == 0 || !StringHelper.IsNumeric(str))
```

Comment:
Can you also check that the length of the string is divisible by 3

Reply:
Sure, done